### DISCUSSION TITLE
How to fetch with tauri's js http api using proxy?

### DISCUSSION THREAD

**Discussion by allala0** [tauri-apps/tauri#6450] [Q&A] [answered]
I want to send a http request using proxy, is there any way to do this with tauri's http api?
Something simmilar to Python requests library: requests.get(url="http://www.google.com", proxies={"http": "address:port"})

**Comment by FabianLars** [comment, marked as answer]
There does not exist such a setting for neither the browser native fetch (<https://github.com/tauri-apps/tauri/issues/4263>) nor tauri's own http client. Feell free to open a feature request for the latter! i don't think there is one yet.
